packages/cdktf-cli/lib/dependencies/dependency-manager.ts

```diff
@@ -241,10 +248,14 @@ export class DependencyManager {
   /**
    * Converts an NPM package name of a pre-built provider package to the name in the target language
    */
-  private convertPackageName(name: string): string {
+  private convertPackageName(name: string, repository: string): string {
     const providerName = name.replace("@cdktf/provider-", "");
     switch (this.targetLanguage) {
       case Language.GO: // e.g. github.com/hashicorp/cdktf-provider-opentelekomcloud-go/opentelekomcloud
+        if (repository) {
+          return `${repository}-go/${providerName}`;
+        }
+
         return `github.com/hashicorp/cdktf-provider-${providerName}-go/${providerName}`;
```

Comment:
I'm torn on whether we should fallback to the old one here. My hunch is that in the future Projen might introduce a change that changes the repository url format in the package.json which might cause this to break – a bit far fetched, I know 😁
But I can also see the opposite where we don't know whether we always had the repository in the package.json in the right format 🤔
I have no idea here, feel free to just discard this – I think yours is safer for today 😅
```suggestion
        return `github.com/cdktf/cdktf-provider-${providerName}-go/${providerName}`;
```

Reply:
Yeah, this is mostly just a 'if something's gone _really_ wrong' kinda thing. That being said, we're probably safe that it won't happen right now, so your change makes more sense.